tests/test_raid.py

```diff
@@ -46,17 +49,23 @@ def test_raid(self, run_command, fs):
 
         res = check_raid_pool(None, None)
         assert json.loads(res) == json.loads(expected)
-        run_command.assert_called_once_with(['mdadm', '--detail', '/dev/md127'])
+        run_command.assert_called_once_with(MDADM_DETAIL_CMD)
 
     def test_raid_error(self, run_command, fs):
         run_command.side_effect = Exception('Error!')
 
         with pytest.raises(XenAPIPlugin.Failure) as e:
             check_raid_pool(None, None)
-        run_command.assert_called_once_with(['mdadm', '--detail', '/dev/md127'])
+        run_command.assert_called_once_with(MDADM_DETAIL_CMD)
         assert e.value.params[0] == '-1'
         assert e.value.params[1] == 'Error!'
 
+    def test_raid_missing(self, run_command, fs):
```

Comment:
The test jobs will fail if we merge this now:
* 8.2 doesn't have the fixed plugin
* I don't think we have updated it in 8.3 yet either

Suggested course of action:
* update the test to only run on 8.3+, or better yet mark it XFAIL on 8.2
* update the xcp-ng-xapi-plugins RPM on 8.3
* and only then, merge this PR

Reply:
Nevermind my comment: I thought we were on xcp-ng-tests